`make` fails due to missing ECMAScript references

### What is the issue with the Web IDL Standard?

Repro:

1. Clone the repository
2. `make remote`
3. Observe:
```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  664k    0   223  100  663k     35   106k  0:00:06  0:00:06 --:--:--    67

Error running preprocessor, returned code: 2.
LINK ERROR: No 'idl' refs found for '%IteratorPrototype%'.
{{%IteratorPrototype%}}
 ✘  Did not generate, due to errors exceeding the allowed error level.
make: *** [Makefile:5: remote] Error 22
```

Also reproduced with a local bikeshed install from pip.

```
(vevn) andrew@Andrew-Workstation:~/whatwg/webidl$ which bikeshed
/home/andrew/whatwg/webidl/vevn/bin/bikeshed
(vevn) andrew@Andrew-Workstation:~/whatwg/webidl$ bikeshed --version
4.2.1
(vevn) andrew@Andrew-Workstation:~/whatwg/webidl$ make local
bikeshed spec index.bs index.html --md-Text-Macro="COMMIT-SHA LOCAL-COPY"
LINK ERROR: No 'idl' refs found for '%IteratorPrototype%'.
{{%IteratorPrototype%}}
 ✘  Did not generate, due to errors exceeding the allowed error level.
make: *** [Makefile:19: local] Error 2
```

--- a/index.bs
+++ b/index.bs
@@ -12528,7 +12528,7 @@ prototype for [=default iterator objects=]
 for the interface.
 
 The \[[Prototype]] [=/internal slot=] of an [=iterator prototype object=]
-must be {{%IteratorPrototype%}}.
+must be {{%Iterator.prototype%}}.
 
 <div algorithm>
   The <dfn>iterator result</dfn> for a [=value pair=] |pair| and a kind |kind| is given by the

